zproject/test_settings.py

```diff
@@ -223,7 +223,7 @@ def set_loglevel(logger_name: str, level: str) -> None:
 }
 
 RATE_LIMITING_RULES = {
-    'api_by_user': [],
+    'api_by_user': [(60, 200), ],
```

Comment:
What's the story behind the change here?

Reply:
@timabbott while running tests, the `RATE_LIMITING_RULES` from test_settings are used. So, it was causing index out of bound exception while trying to access `rules['api_by_user'][0][0]`. Though, I am going to use `garbage_collector` instead of `ttl_cache` so I think these changes won't be required anymore.